tests/test/libretro-test.c

```diff
@@ -8,9 +8,19 @@
 
 #include "libretro.h"
 
-static uint32_t *frame_buf;
+typedef struct core_data_t {
```

Comment:
I think to really demonstrate this envcall, you'd need to move _all_ currently-`static` state into it

Reply:
That would be awesome. I'll see how much can be moved. Some of the callbacks are called before the core is initialized.